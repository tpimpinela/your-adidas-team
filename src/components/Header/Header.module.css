.header {
  display: grid;
  place-items: center;
  border-bottom: 1px solid var(--border-color);

  & h1 {
    display: flex;
    align-items: center;
    gap: 0.4em;
    padding: 1.2rem 0;
    margin: 0;
  }

  & .header__logo {
    width: 80px;
  }
}
