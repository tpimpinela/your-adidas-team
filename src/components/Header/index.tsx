import styles from "./Header.module.css";
import adidasLogo from "../../assets/adidas-logo.png";

const Header = () => (
  <header className={styles.header}>
    <h1>
      Your
      <img className={styles.header__logo} src={adidasLogo} />
      team
    </h1>
  </header>
);

export default Header;
