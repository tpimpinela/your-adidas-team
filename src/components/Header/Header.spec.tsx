import { beforeEach, describe, expect, it } from "vitest";
import { render, RenderResult } from "@testing-library/react";
import Header from ".";

describe("<Header />", () => {
  let renderResult: RenderResult;

  beforeEach(() => {
    renderResult = render(<Header />);
  });

  it("should match the snapshot", () => {
    expect(renderResult.container).toMatchSnapshot();
  });
});
