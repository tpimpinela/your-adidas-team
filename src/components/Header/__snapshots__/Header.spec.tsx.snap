// Vitest Snapshot v1

exports[`<Header /> > should match the snapshot 1`] = `
<div>
  <header
    class="_header_63c2ea"
  >
    <h1>
      Your
      <img
        class="_header__logo_63c2ea"
        src="/src/assets/adidas-logo.png"
      />
      team
    </h1>
  </header>
</div>
`;
